website/meta/universe.json

```diff
@@ -231,6 +231,49 @@
                 "website": "https://koaning.io"
             }
         },
+        {
+            "id": "tokenwiser",
+            "title": "tokenwiser",
+            "slogan": "Connect vowpal-wabbit & scikit-learn models to spaCy to run simple classification benchmarks. Comes with many utility functions for spaCy pipelines.",
+            "github": "rasahq/tokenwiser",
```

Comment:
The repo doesn't exist.

Reply:
d0h! My bad! I copied from a previous package I submitted that was hosted by my employer. This one is hosted on my private github [here](https://koaning.github.io/tokenwiser/index.html). Fixing!